src/layers/domain/core/load_questionnaire.py

```diff
@@ -0,0 +1,52 @@
+from pathlib import Path
+
+from domain.core import questionnaire_validation_custom_rules
+from event.json import json_load
+
+from .questionnaire import ALLOWED_QUESTION_TYPES, Questionnaire
+
+PATH_TO_HERE = Path(__file__).parent
+
+
+def load_json_file(file_path):
+    with open(file_path, "r") as file:
+        data = json_load(file)
+    return data
+
+
+def convert_answer_type_names_to_python_types(answer_types):
+    answer_types = set(
+        _type for _type in ALLOWED_QUESTION_TYPES if _type.__name__ in answer_types
+    )
+
+    return answer_types
+
+
+def convert_rule_names_to_rule_functions(validation_rules):
+    return [
+        getattr(questionnaire_validation_custom_rules, rule)
+        for rule in validation_rules
+    ]
+
+
+def render_questionnaire(questionnaire_name, questionnaire_version):
+    json_file_path = f"{PATH_TO_HERE}/questionnaires/{questionnaire_name}/v{questionnaire_version}.json"
```

Comment:
Do we want to validate the questionnaire names can only be spine_deivce and spine_endpoint?